src/RadForce/test_radiation_force.cpp

```diff
@@ -258,7 +239,7 @@ auto problem_main() -> int
 	for (int n = 0; n < nvars; ++n) {
 		// for x-axis:
 		BCs_cc[n].setLo(0, amrex::BCType::ext_dir);
-		BCs_cc[n].setHi(0, amrex::BCType::ext_dir);
+		BCs_cc[n].setHi(0, amrex::BCType::foextrap); // extrapolate
```

Comment:
Using `foextrap` makes the problem mathematically ill-posed. Can you try using `NSCBC::setOutflowBoundary` or `NSCBC::setOutflowBoundaryLowOrder`: https://github.com/quokka-astro/quokka/blob/4ab068ebda59e45ab21eccb05aae17985d570548/src/NSCBC_outflow.hpp#L358?

Reply:
This has to called from within `setCustomBoundaryConditions` above.